pkg/ensign/interceptors/auth.go

```diff
@@ -154,3 +156,17 @@ func (a *Authenticator) isPublic(route string) bool {
 	_, ok := a.publicRoutes[route]
 	return ok
 }
+
+func UserIP(addr net.Addr) string {
+	// If this is a TCP ip address then handle it directly
+	if tcpaddr, ok := addr.(*net.TCPAddr); ok {
+		return tcpaddr.IP.String()
+	}
+
+	// Try parsing the IP address the hard way
+	ipaddr := addr.String()
+	if host, _, err := net.SplitHostPort(ipaddr); err == nil {
+		return host
+	}
+	return ipaddr
+}
```

Comment:
We were getting the following error in sentry; so this code strips off the port of the IP address.

![Screenshot 2023-10-18 at 08 06 36](https://github.com/rotationalio/ensign/assets/745966/e2e2a57a-6077-4105-9b2d-fe2f84d3b4d4)